pkg/apis/v1alpha1/common.go

```diff
@@ -191,3 +192,22 @@ type FieldSelectorRequirement struct {
 	// the values array must be empty.
 	Values []string `json:"values,omitempty"`
 }
+
+type RealizedResource struct {
```

Comment:
I think it'd be nice to have a description for these , e.g., for the others it's pretty well defined

```console
$ k explain workload.status
KIND:     Workload
VERSION:  carto.run/v1alpha1
...
   observedGeneration   <integer>
     ObservedGeneration refers to the metadata.Generation of the spec that
     resulted in the current `status`.

   resources    <[]Object>
     Resources contain references to the objects created by the Supply Chain and
     the templates used to create them. It also contains Inputs and Outputs that
     were passed between the templates as the Supply Chain was processed.
```

but then lacking at this level

```console
$ k explain workload.status.resources
KIND:     Workload
VERSION:  carto.run/v1alpha1

RESOURCE: resources <[]Object>

DESCRIPTION:
     Resources contain references to the objects created by the Supply Chain and
     the templates used to create them. It also contains Inputs and Outputs that
     were passed between the templates as the Supply Chain was processed.

FIELDS:
   inputs       <[]Object>

   name <string> -required-

   outputs      <[]Object>
```

Reply:
We did it. But I'm sold that anyone will understand what we are talking about.